cves/kernel/CVE-2021-38198.yml

```diff
@@ -171,10 +173,11 @@ discovered:
 
     If there is no evidence as to how this vulnerability was found, then please
     explain where you looked.
-  answer:
-  automated:
-  contest:
-  developer:
+  answer: Looked in kernel, bugzilla, etc. No word of reported by, or anything of the likes. There were 
```

Comment:
Here's where your yaml syntax is broken - for multiline strings you need the | format thing like the others